web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -34,8 +38,21 @@ const sxs = {
 };
 
 const Dashboard = (): JSX.Element => {
+  const { wallet } = useWallet();
   const theme = useTheme();
   const isIssuer = useQueryIfIssuer();
+  const isCreditClassCreator = useQueryIfCreditClassCreator();
+  const classes = useQueryListClasses();
+  const adminClasses = classes?.classes.filter(
+    x => x.admin === wallet?.address,
+  );
+  let isCreditClassAdmin: boolean;
+  if (adminClasses) {
+    isCreditClassAdmin = adminClasses.length > 0;
+  } else {
+    isCreditClassAdmin = false;
+  }
+  const creditClassTabHidden = !(isCreditClassCreator || isCreditClassAdmin);
```

Comment:
if the person is neither a credit class creator or a credit class admin, we will mark the tab as "hidden"